src/facade/dragonfly_connection.cc

```diff
@@ -2075,6 +2098,16 @@ void Connection::DecrNumConns() {
     --stats_->num_conns_other;
 }
 
+bool Connection::IsReplySizeOverLimit() const {
+  std::atomic<size_t>& reply_sz = tl_facade_stats->reply_stats.squashing_current_reply_size;
+  size_t current = reply_sz.load(std::memory_order_acquire);
+  const bool over_limit = reply_size_limit != 0 && current > 0 && current > reply_size_limit;
+  LOG(INFO) << "current: " << current << "/" << reply_size_limit;
+  VLOG_IF(2, over_limit) << "MultiCommandSquasher overlimit: " << current << "/"
```

Comment:
Actually if we are over limit we want to see this in logs, make this a warning and print once a second